README.md

```diff
@@ -36,3 +36,6 @@ This step-by-step guide assumes you have [NodeJS](https://nodejs.org/en/) versio
 Coming soon :tm:
 
 [Join the Discord](https://needle.gg/chat) if interested!
+
+## License
+This program is released under [version 3 of the GNU Affero General Public License](LICENSE).
```

Comment:
```suggestion
    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU Affero General Public License as published by
    the Free Software Foundation, version 3.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU Affero General Public License for more details.

    You should have received a copy of the GNU Affero General Public License
    along with this program.  If not, see <https://www.gnu.org/licenses/>.
```